oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxHWDiskStore.java

```diff
@@ -270,6 +300,27 @@ private static void computeDiskStats(LinuxHWDiskStore store, String devstat) {
         store.transferTime = devstatArray[UdevStat.ACTIVE_MS.ordinal()];
     }
 
+    private static String getPartitionNameForDmDevice(String vgName, String lvName) {
+        return new StringBuilder().append(DEV_LOCATION).append("/").append(vgName).append("/").append(lvName).toString();
```

Comment:
Minor point: some of our CI will say the append will be more efficient with a character rather than single character string, so I'd suggest single quotes on the slashes here.  Also if you edit DEV_LOCATION to include the trailing slash you won't need it here.

Reply:
noted.